charts/barman/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: barman
 type: application
 description: Chart for Barman PostgreSQL Backup and Recovery Manager
-version: 0.1.1
+version: 0.2.0
```

Comment:
@hairmare this needs some changes in our argocd [project definition](https://git.adfinis.com/ad-sy/helm-infrastructure-int-aks/-/blob/master/argocd/aks-adfch-k8s-int-prod-02/app-barman.yaml#L20) as well. same thing [for test](https://git.adfinis.com/ad-sy/helm-infrastructure-int-aks/-/blob/master/argocd/aks-adfch-k8s-int-test-02/app-barman.yaml#L20) too